tests/Loader/LoaderIntegrationTest.php

```diff
@@ -2575,5 +2575,29 @@ public function provideFixturesToGenerate()
                 ],
             ],
         ];
+
+        yield 'method call reference value' => [
```

Comment:
could you check if there is a test for a method taking a parameter? If not I think it's worth adding as well in that PR